contribs/gmf/apps/desktop/index.html

```diff
@@ -250,8 +250,8 @@
     <script src="../../../../node_modules/angular-gettext/dist/angular-gettext.js"></script>
     <script src="../../../../node_modules/angular-dynamic-locale/dist/tmhDynamicLocale.js"></script>
     <script src="../../../../node_modules/bootstrap/dist/js/bootstrap.js"></script>
-    <script src="../../../../node_modules/d3/d3.min.js"></script>
     <script src="../../../../node_modules/file-saver/FileSaver.min.js"></script>
+    <script src="../../../../node_modules/d3/build/d3.min.js" type="text/javascript"></script>
```

Comment:
` type="text/javascript"` is the default and only possible value => better to don't add it ...